HACKING.md

```diff
@@ -9,21 +9,22 @@ Overview of Mender client architecture.
                                           |   |
                                           |   |
                                           |   |
-         +------------------+     +-------+---v--------+     +-------------------+
-         |      daemon      |     |      client        |     |      mender       |
-         |                  |     |                    |     |                   |
-         +------------------+     +--------------------+     +-------------------+
-         |                  |     | Updater            |     | Controller        |
-         |                  |     |                    |     |                   |
-         +------------------+     +--------------------+     +-------------------+
-         | NewDaemon        |     | NewUpdater         |     | NewMender         |
-         | Run              |     | NewHttpsClient     |     | GetState          |
-         | StopDaemon       <-----+ NewHttpClient      |     | GetCurrentImageID |
-         |                  |     | GetScheduledUpdate |     | LoadConfig        |
-         |                  |     | FetchUpdate        |     | GetUpdaterConfig  |
-         |                  |     | Bootstrap          |     | GetDaemonConfig   |
-         |                  |     |                    |     |                   |
-         +-------------^-^--+     +--------------------+     +-----+---^---------+
+         +------------------+     +-------+---v--------+     +------------------------+
+         |      daemon      |     |      client        |     |      mender            |
+         |                  |     |                    |     |                        |
+         +------------------+     +--------------------+     +------------------------+
+         |                  |     | Updater            |     | Controller             |
+         |                  |     |                    |     |                        |
+         +------------------+     +--------------------+     +------------------------+
+         | NewDaemon        |     | NewUpdater         |     | NewMender              |
+         | Run              |     | NewHttpsClient     |     | GetState               |
+         | StopDaemon       <-----+ NewHttpClient      |     | GetCurrentArtifactName |
+         |                  |     | GetScheduledUpdate |     | LoadConfig             |
+         |                  |     | FetchUpdate        |     | GetUpdaterConfig       |
+         |                  |     | Bootstrap          |     | GetDaemonConfig        |
+         |                  |     |                    |     | GetDeviceType          |
+         |                  |     |                    |     |                        |
+         +-------------^-^--+     +--------------------+     +-----+---^--------------+
```

Comment:
I think we should remove this file for now. It is outdated and we are not updating it.